apps/api/src/services/vote-manager.service.ts

```diff
@@ -337,7 +337,7 @@ export class VoteManager {
     const now = new Date();
 
     // If at least one voting date is not set, voting for the comp is disabled
-    if (!competition.votingStartDate && !competition.votingEndDate) {
+    if (!competition.votingStartDate || !competition.votingEndDate) {
```

Comment:
this simple changed affected a lot the tests. that's why a bunch of tests changes